tabbycat/motions/statistics.py

```diff
@@ -236,16 +244,17 @@ def _collate_prelim_motion_annotations(self):
             motion.counts_by_bench = {'gov': 0, 'opp': 0}
 
             for side in self.tournament.sides:
+                side_abbrv = abbrv_of_bp_side(side)
```

Comment:
Should use `tournaments.utils.get_side_name` like:

```py
side_abbrev = get_side_name(side, 'abbr')
```

Then we wouldn't need the new function.

Reply:
Thanks! I've updated it to use that function.

I don't believe it affects two-team formats (e.g. here this problem doesn't seem to exist https://edudrift.calicotab.com/australs2024/motions/statistics/).